aldryn_faq/tests/test_base.py

```diff
@@ -6,17 +6,20 @@
     from importlib import reload
 import random
 import string
+import sys
 
 from django.conf import settings
 from django.contrib.auth import get_user_model
 from django.contrib.sites.models import Site
 from django.test import override_settings, RequestFactory, TransactionTestCase
 from django.utils.translation import override
+from django.core.urlresolvers import clear_url_caches
 
 from cms import api
 from cms.models import Page, Title
 from cms.utils.conf import get_cms_setting
 from cms.utils.i18n import get_language_list
+from cms.appresolver import clear_app_resolvers
```

Comment:
please add this import under the api import

Reply:
Updated